README.md

```diff
@@ -150,6 +150,14 @@ DdRum.startResource('<res-key>', 'GET', 'http://www.example.com/api/v1/test', ne
 //…
 DdRum.stopResource('<res-key>', 200, 'xhr', new Date().getTime(), {});
 ```
+
+## Resource Timings
+
+Resource tracking is able to provide the following timings:
+
+* `First Byte` - this the time between the request is scheduled to be sent and response is first seen. This includes time for the request preparation on the native level, network latency and time it took the server to prepare the response.
```

Comment:
```suggestion
* `First Byte` - The time between the scheduled request and the first response. This includes time for the request preparation on the native level, network latency, and the time it took the server to prepare the response.
```

Reply:
@ruthnaebeck I think suggestion should be update here. Suggestion has 2 issues:

* With a suggestion it has a meaning that start of the timing is after the request is sent, which is not the case. The timing start of the timing is really before request is even scheduled.
* We can have only one response (even if it has multiple chunks). Maybe we can use phrase `first byte of the response`?

Reply:
updated, let me know if that works

Reply:
thanks, works great!